src/main/java/io/ebean/dbmigration/ddlgeneration/platform/BaseTableDdl.java

```diff
@@ -832,22 +852,11 @@ protected void alterTableDropColumn(DdlBuffer buffer, String tableName, String c
   }
 
   protected void alterTableAddColumn(DdlBuffer buffer, String tableName, Column column, boolean onHistoryTable) throws IOException {
-
-    String convertedType = platformDdl.convert(column.getType(), false);
-    buffer.append("alter table ").append(tableName)
-      .append(" add column ").append(column.getName())
-      .append(" ").append(convertedType);
-
-    if (!onHistoryTable) {
-      if (isTrue(column.isNotnull())) {
-        buffer.append(" not null");
-      }
-      if (hasValue(column.getCheckConstraint())) {
-        buffer.append(" constraint ").append(column.getCheckConstraintName());
-        buffer.append(" ").append(column.getCheckConstraint());
-      }
+    String ddl = platformDdl.alterTableAddColumn(tableName, column, onHistoryTable);
```

Comment:
sqlserver has its own alter syntax: 
Mysql/Postgres/H2: "alter table add column <colname>"
Sqlserver: "alter table add <colname>"
so I redirected this to platformDdl